docs/volunteering/internal-teams-guidance.md

```diff
@@ -0,0 +1,72 @@
+# Guidance for internal teams
+
+Student Robotics has a number of internal teams which contribute to its overall
+mission. The following is guidance based on experience within the Competition
+Team (and its predecessor the Core Team), but which any or all such teams may
+find useful.
+
+## Communication
+
+Over communicate. Both between yourselves and others. It's very easy to assume
+that you've mentioned something to other people and that they therefore know
+what you're working on/towards/etc., yet it later turn out that their
+understanding is very different to yours. Over communication doesn't eradicate
+this, but it does greatly reduce it.
+
+### Have regular short meetings
+
+Establish a regular cadence of _short_ meetings. The goal of these should be
+semi-formalised *very high level* progress check-ins to keep on track, provide
+an opportunity to unblock things, get input on challenges and raise concerns.
+
+Be aware that these meetings inherently exclude others, so should focus on
+things which either only the Team can do or which (for whatever reason) _must_
+remain their direct responsibility. (For clarity: things which merely _happen_
+to be their direct responsibility don't count)
+
+### Have regular updates to others
+
+Establish a regular cadence of updates to volunteers. The goal here is to
+provide a way for volunteers to jump in to ongoing things and to celebrate
+achievements.
+
+## Encourage input from others
+
+Encourage input from a wide team of volunteers. While mass emails can work,
+you'll also need to approach people individually about things. Try to meet new
+volunteers in person as much as possible - it's _much_ easier to work remotely
+with people you've met in person and this also helps build trust which allows
+you to more easily work through misunderstandings.
+
+## Documentation
+
+Use the existing documentation. If you've found these docs, then you're already
+doing well, if not then how are you reading this‽
+
+Update the documentation. The existing docs are guidelines, not rules. The best
+way to help your future selves (and ensure you understand the thing you're
+working on) is to improve the docs. Note that the docs include a task list for
+running the competition -- use that too!
+
+## Project planning
+
+Kick-off projects. At the start of a project or task, ensure that those who are
+going to be doing the work understand what's involved and that everyone has a
+clear picture of what a good outcome looks like. I realise this totally sounds
```

Comment:
The voice of the page changes quite a lot with the use of "I"

```suggestion
clear picture of what a good outcome looks like. This may totally sound
```

Reply:
yeah, I wasn't sure about this. ac8dcc6 changes it.